webapp/src/components/Metrics/PerformanceAnalysisTable.tsx

```diff
@@ -0,0 +1,658 @@
+            ),
+          },
+        ]
+      ),
+      ...pipelines.map<Column<Row>>((pipeline) => ({
+        ...METRIC_COLUMN,
+        field: `${pipeline}ECE`,
+        ...groupHeader(pipeline, "ECE"),
+        align: "right",
+        description: ECE_TOOLTIP,
+        valueGetter: ({ row }) => row[pipeline]?.ece,
+        renderCell: ({ value }: GridCellParams<number | undefined>) =>
+          value !== undefined && (
+            <VisualBar
+              formattedValue={value.toFixed(2)}
+              value={value}
+              color={(theme) => alpha(theme.palette.primary.dark, 0.5)}
+            />
+          ),
+      })),
+      {
+        ...DELTA_METRIC_COLUMN,
+        field: `deltaECE`,
+        headerName: "Delta",
+        headerClassName: "no-border-right",
+        cellClassName: "border-left",
+        headerAlign: "center",
+        valueGetter: ({ row }) =>
+          row.comparedPipeline &&
+          row.comparedPipeline.ece - row.basePipeline.ece,
+        renderCell: ({ value }: GridCellParams<number>) => (
+          <DeltaComputationBar
+            value={value}
+            formattedValue={value && (value as number).toFixed(2)}
+            width={isNaN(value) ? 0 : Math.abs(value) * 50}
+          />
+        ),
+      },
+    ];
+  }, [
+    comparedPipeline,
+    config,
+    metricInfo,
+    pipeline,
+    selectedMetricPerFilterOption,
+    sortModel,
+  ]);
+
+  React.useEffect(() => {
+    if (comparedPipeline === pipeline.pipelineIndex) {
+      setComparedPipeline(undefined);
+    }
+    const visibility = comparedPipeline !== undefined;
+    const labelColumnVisibility =
+      comparedPipeline !== undefined &&
+      selectedMetricPerFilterOption !== "label";
+    const columnVisibilityEntries = [
+      "UtteranceCount",
+      ...ALL_OUTCOMES,
+      ...Object.keys(metricInfo ?? {}),
+      "ECE",
+    ].flatMap((field) => [
+      [
+        `comparedPipeline${field}`,
+        field === "UtteranceCount" ? labelColumnVisibility : visibility,
+      ],
+      [
+        `delta${field}`,
+        field === "UtteranceCount" ? labelColumnVisibility : visibility,
+      ],
+    ]);
+    setColumnVisibilityModel(Object.fromEntries(columnVisibilityEntries));
+  }, [pipeline, selectedMetricPerFilterOption, comparedPipeline, metricInfo]);
+
+  const RowLink = (props: RowProps<Row>) => (
+    <Link
+      style={{ color: "unset", textDecoration: "unset" }}
+      to={`/${jobId}/dataset_splits/${selectedDatasetSplit}/performance_overview${constructSearchString(
+        {
+          ...(props.row.id !== OVERALL_ROW_ID && {
+            [selectedMetricPerFilterOption]: [
+              props.row.basePipeline.filterValue,
+            ],
+          }),
+          ...pipeline,
+        }
+      )}`}
+    >
+      <GridRow {...props} />
+    </Link>
+  );
+
+  const LabelHeader = () => (
+    <Box
+      position="relative"
+      height={LABEL_HEIGHT}
+      width={LABEL_WIDTH}
+      top={LABEL_HEIGHT}
```

Comment:
Instead of offsetting the content with `top`, leaving a blank space above the table, I would use a negative `marginBottom`, like that:
```suggestion
      marginBottom={-DATA_GRID_PROPS_DEFAULT_VALUES.headerHeight}
```

Reply:
Perfect!. margin-bottom is more suitable, but it also needed to add padding for the box around the table.

Reply:
Also, `LABEL_HEIGHT` is quite a bad name, since the select can be label, prediction and smart tags, not just label.

Reply:
Right!.. I believe we should have one name for the select column.

Reply:
Considering the "DATA_GRID_PROPS_DEFAULT_VALUES.headerHeight" being constant already, I removed declaring already with const again. So there is no LABEL_HEADER in the code.

Reply:
The values in that column are generally called "filter values". The field is `field: "filterValue"`. For reference, the type of each row is `MetricsPerFilterValue`.

Reply:
Additionally, as the label name does not deliver a good message here, I changed the "LABEL_WIDTH" as "HEADER_WIDTH" as this CSS applies to header.